src/commons/assessment/AssessmentTypes.ts

```diff
@@ -121,7 +121,7 @@ export interface IContestVotingQuestion extends BaseQuestion {
   prepend: string;
   postpend: string;
   contestEntries: ContestEntry[];
-  contestLeaderboard: ContestEntry[];
+  scoreLeaderboard: ContestEntry[];
```

Comment:
This is the type of the JSON returned from the backend right, or am I mistaken? If it's the case, then will there be a backend PR to accompany this?

Reply:
Yes, we'll make sure to start working on that. Thanks for noticing!

Reply:
Ok, thanks! In the meantime I'll just mark this PR as blocked so it doesn't get accidentally merged.